### PR TITLE
Refactor to remove internal utility `getOsEol()`

### PR DESCRIPTION
**Pull request by ybiquitous**
> Which issue, if any, is this issue related to?

Ref https://github.com/stylelint/jest-preset-stylelint/pull/66

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -203,7 +202,7 @@
     "eslint-plugin-jest": "^27.2.3",
     "husky": "^8.0.3",
     "jest": "^29.6.1",
-    "jest-preset-stylelint": "^6.1.0",
+    "jest-preset-stylelint": "github:stylelint/jest-preset-stylelint#getOsEol-not-needed",
```
Comment (ybiquitous) [inline package.json:205]: - [ ] Wait for a new version of `jest-preset-stylelint`

**Review by jeddy3 — COMMENTED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (ybiquitous) [reply package.json]: Updated to https://github.com/stylelint/jest-preset-stylelint/releases/tag/6.1.1 via 02ed7173b

**Review by jeddy3 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**